entrypoint.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+
+URI=https://api.github.com
+API_HEADER="Accept: application/vnd.github.v3+json"
+AUTH_HEADER="Authorization: token $GITHUB_TOKEN"
+git config --global --add safe.directory /github/workspace
```

Comment:
Required otherwise github will moan about it and cause the action to fail.

Must be done before other git actions